core/templates/core/user_accounts/password_change/password_change_form.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'core/base.html' %}
+{% load i18n static %} <!-- Serves translation -->
+
+{% block title %}
+  Squire - Password Change
```

Comment:
Everything else in this template is translated, isn't it more logical to translate the title as well?

Reply:
See the comment at https://github.com/EricTRL/Squire/pull/25#discussion_r326639283